actionpack/lib/action_controller/railtie.rb

```diff
@@ -112,7 +112,9 @@ class Railtie < Rails::Railtie # :nodoc:
         app.config.action_controller.log_query_tags_around_actions
 
       if query_logs_tags_enabled
-        app.config.active_record.query_log_tags += [:controller, :action]
+        [:controller, :action].each do |tag|
+          app.config.active_record.query_log_tags << tag unless app.config.active_record.query_log_tags.include?(tag)
+        end
```

Comment:
```suggestion
        app.config.active_record.query_log_tags |= [:controller, :action]
```

Would be simpler, no?

Reply:
Haha yes it would.